lalrpop/Cargo.toml

```diff
@@ -33,26 +33,21 @@ term = { version = "0.7", default_features = false }
 tiny-keccak = { version = "2.0.2", features = ["sha3"] }
 unicode-xid = { version = "0.2", default_features = false }
 
-# This dependency is only needed for binary builds, if you use LALRPOP as
-# library, disable it in your project by setting default_features = false.
```

Comment:
As I remember, this is related to `cargo install`. By removing this feature from default feature, `cargo install lalrpop` will fail and have to be `cargo install lalrpop --features pico-args`. 
https://github.com/rust-lang/cargo/issues/10409

I'd like to keep this feature as default unless cargo add a feature or we make a new crate `lalrpop-cli`

Reply:
It will "succeed" (exit with 0) without building/installing the lalrpop binary and display:

```
warning: none of the package's binaries are available for install using the selected features
  bin "lalrpop" requires the features: `pico-args`
Consider enabling some of the needed features by passing, e.g., `--features="pico-args"`
```

Reply:
One day maybe cargo will get https://github.com/rust-lang/rfcs/pull/3374